demos/pager/css/pager.css

```diff
@@ -78,12 +74,11 @@
 
 .demo-pager-pageLink-default a {
     /* Using display:block and float:left, it creates helps create a horizontal list with fixed width elements.*/
```

Comment:
While you're making changes in this file, could you please correct the comment here by removing the extra "creates"?